code/_globalvars/lists/mobs.dm

```diff
@@ -20,6 +20,7 @@ GLOBAL_LIST_EMPTY(ai_list)
 GLOBAL_LIST_EMPTY(pai_list)
 GLOBAL_LIST_EMPTY(available_ai_shells)
 GLOBAL_LIST_EMPTY(simple_animals)
+GLOBAL_LIST_EMPTY(spidermobs)				//all spider mobs

```

Comment:
these are only logged in spider mobs, fix the comment or change the functionality

Reply:
preferably the latter

Reply:
Its only use is to message players [as of now at least] , it should only be Player-controlled spiders.

Reply:
Yes, but it's much saner to keep track of all spiders and who cares if AI mobs get the message?

Reply:
fair enough

Reply:
i'll get around to this zzz